src/views/Team/Outlet.vue

```diff
@@ -112,30 +112,32 @@
                 </ion-button>
               </div>
 
-              <h2>{{ $t("team.subteams.title") }}</h2>
-              <ul v-if="subteams" class="subteams" data-cy="subteams">
-                <li v-for="t in subteams" :key="t.id">
-                  <router-link
-                    :to="{ name: 'ViewTeam', params: { teamSlug: t.slug } }"
-                  >
-                    <ion-chip outline>
-                      <avatar withName :profile="t" size="1.5em" />
-                    </ion-chip>
-                  </router-link>
-                </li>
-
-                <ion-chip outline>
+              <div>
+                <h2 v-if="subteams?.length || canEdit">
```

Comment:
isn't an empty array falsy in vue - in any case it is computed and 'soft' guaranteed to be an array so `?` is unneessary? Either way, I consider it better style to have any non-trivial checks outsourced into a `computed` on the component for legibility.

Reply:
AFAIK an empty array is not falsy in vue. (same as with javascript). The `?` may be unnecessary, but it's more failsafe to use it. ;)

Also personally I like simple (but non-trivial) checks more directly in the template. The code is faster to read because I see directly what happens. But I'll change that to your preferences.